pom.xml

```diff
@@ -55,6 +55,11 @@
             <artifactId>mockito-all</artifactId>
             <version>1.9.5</version>
         </dependency>
+        <dependency>
+            <groupId>com.google.apis</groupId>
+            <artifactId>google-api-services-gmail</artifactId>
```

Comment:
extreme nit: maybe put this in the feature/gmail-utility branch instead

Reply:
Agreed - Fixed!